index.html

```diff
@@ -8,23 +8,16 @@
     <title>Mobile Portoflio</title>
 </head>
 <body>
-    <div class="container">
         <header>
-            <h1 class="logo">Ngala</h1>
-            <menu>
-                <ul class="menu">
-                    <li class="mylist"><a href="#">Portoflio</a></li>
-                    <li class="mylist"><a href="#">About</a></li>
-                    <li class="mylist"><a href="#">Contact</a></li>
-                </ul>
-            </menu>
+            <a class="logo" href="#">Ngala</a>
             <nav>
                 <div class="mybar"></div>
                 <div class="mybar"></div>
                 <div class="mybar"></div>
                 
             </nav>
         </header>
+        <main>
         <section>
         <div class="background_image">
             <h1 class="header">I’m Ngala Mac
```

Comment:
- [ ] Kindly 😊 try to decrease the line height of this heading as you see here the line height is greater than the Figma Design, so let's decrease it to make it nice like the Figma Design 😃🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   ![image](https://github.com/ngalamac/mobile-portfolio/assets/83360636/b4cba945-cf47-43bf-a1b0-c921b2f72698)  |     ![image](https://github.com/ngalamac/mobile-portfolio/assets/83360636/3bfa8b0c-f46f-4e6f-a11d-60444bdbd05f)  |